utils/rpm.mk

```diff
@@ -68,7 +69,7 @@ submodules:
 ${subcomps}:
 	make -C ${PBENCHTOP}/$@ DESTDIR=${TBDIR}/$@ install
 
-$(RPMSRPM)/$(prog)-$(version)-$(seqno)$(sha1).src.rpm: srpm
+$(RPMSRPM)/$(prog)-$(version)-$(seqno)g$(sha1).src.rpm: srpm
```

Comment:
Make variable names are case-sensitive...so where does `$(version)` get defined?  (Should that be `$(VERSION)`, here and at line 82?  Note that the references in lines 82 and 83 don't match.)

Reply:
There is a comment in the rpm.mk on b0.69, saying "version and sha1 have to be provided by the including Makefile" - although the b0.69 Makefile does not do that either.

Reply:
All I can say is that since the same mistake was made in two different places (the "g" as well as the undefined version), it all came out right in the end :-) Just a bit of unnecessary rebuilding...

Reply:
OK, so the targets `copr` and `copr-test` (and, soon, `copr-ci`) depend on `$(RPMSRPM)/$(prog)-$(version)-$(seqno)$(sha1).src.rpm` (to which Peter is adding a `g`) which in turn depends on the `srpm` target which, after removing `${RPMSRPM}/$(prog)-*.src.rpm` (which is probably too broad a wildcard...), invokes `rpmbuild -bs ${RPMSPEC}/${prog}.spec` which apparently produces `$(RPMSRPM)/$(prog)-$(VERSION)-$(seqno)g$(sha1).src.rpm` (which already has a `g` and a `$(VERSION)` component in its name).

So, without Peter's change, it all works because the `copr-cli` command finds the SRPM which `rpmbuild` produced despite the lack of a `$(version)` definition (and the missing `g`).  Peter's change (plus the up-casing of the `$(version)` reference) might allow Make to skip over rebuilding the SRPM if Make thinks that it's up-to-date; without those changes, I imagine that the process will continue to work as it has done, rebuilding the SRPM every time because Make can never find the one that we specified.  😈

(So, I'm not convinced that just adding the `g` is sufficient to actually change anything.)

Reply:
I believe you are correct.

Reply:
@webbnh, I believe this is now fixed.